dev/NumberBox/NumberBox.idl

```diff
@@ -60,7 +60,12 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String PlaceholderText;
     Windows.UI.Xaml.Controls.Primitives.FlyoutBase SelectionFlyout;
     Windows.UI.Xaml.Media.SolidColorBrush SelectionHighlightColor;
-    Windows.UI.Xaml.TextAlignment TextAlignment;
+
+    [WUXC_VERSION_PREVIEW]
```

Comment:
You also need to include `static Windows.UI.Xaml.DependencyProperty TextAlignmentProperty{ get; };` here.

Reply:
Done, makes sense.